pages/common/git-annotate.md

```diff
@@ -0,0 +1,14 @@
+# git annotate
+
+> Show commit hash and last author on each line of a file.
+> Note: Use `git blame`.
+> This only difference between `git annotate` and `git blame` is that the former has a slightly different output format, and is used only for backwards-compatibility, and provide a more familiar command name for people coming from other SCM systems.
```

Comment:
Perhaps this will be more concise?
```suggestion
> See `git blame`, which is preferred over `git annotate`.
> `git annotate` is provided for those familiar with other version control systems.
```

Reply:
So much better.